src/main/kotlin/cv/nmnb/domain/like/domain/Like.kt

```diff
@@ -0,0 +1,35 @@
+package cv.nmnb.domain.like.domain
+
+import cv.nmnb.domain.post.domain.Post
+import cv.nmnb.domain.user.domain.User
+import cv.nmnb.global.domain.BaseEntity
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import jakarta.persistence.UniqueConstraint
+
+@Entity
+@Table(
+    name = "user_post_likes",
+    uniqueConstraints = [UniqueConstraint(columnNames = ["user_id", "post_id"])],
+)
+class Like(
```

Comment:
코틀린 강의에서 언급한 것처럼 Entity가 생성되는 로직 찾기 쉽도록 constructor 지시자 붙이는 것 어떨까 제안해봅니다!! 
```suggestion
class Like constructor(
```

Reply:
근데.. 결국 DTO의 of를 통해 생성할테니 굳이 안 붙여도 되겠다는 생각이 드네요,,,, 요 코멘트 그냥 무시해주십쇼!!!

Reply:
네네! 요거 혹시 나중에 필요해지면 추가해보는걸로 해요!